components/TableContents.js

```diff
@@ -0,0 +1,54 @@
+import PropTypes from 'prop-types'
+import Link from 'next/link'
+import Links from './Links'
+
+export default function TableContents(props) {
+  const lang = props.lang?.trim?.() === 'fr' ? 'fr' : 'en'
+  const { id, sectionList } = props
+  return (
+    <>
+      <section id={id}>
+        <h2 className="font-display text-2xl leading-[26px] font-bold text-gray-darker">
+          {lang === 'en' ? 'On this page' : 'Sur cette page'}
+        </h2>
+        <ul className="pl-[22px] ml-3.5">
+          {sectionList.map((option, index) => {
+            return (
+              <li
+                key={index}
+                className={
+                  'underline underline-offset-4 text-20px text-gray-darker underline-offset-3.1 decoration-1 pr-1 list-outside list-disc'
+                }
+              >
+                <Links
```

Comment:
This can definitely use next/link instead since you can now apply styling to next/link as it just passes it down to the child `a` element

Reply:
Good Idea.